.gitignore

```diff
@@ -112,6 +112,7 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+.idea/
```

Comment:
This is just an annoying file that gets created when I edit files with my editor and I want to ignore it, hence on the git ignore list